### PR TITLE
Test

### PR DESCRIPTION
**Pull request by petreturcu**

### CODE REVIEW

**Comment by Myatkhine22** [comment]
Please explain what are you commiting.

**Comment by petreturcu** [comment]
The code changes how X does Y because J wants its that way.

File: MyNewFile.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Something
+Test
```
Comment (Myatkhine22) [inline MyNewFile.txt:2]: Are you sure this is what you need to change.

**Review by Myatkhine22 — CHANGES_REQUESTED** [review]
Please add this line.

File: MyNewFile.txt (hunk 1 above)
Comment (petreturcu) [reply MyNewFile.txt:2]: Thank you, I've added the line.

**Review by Myatkhine22 — APPROVED** [review]

**Pull request merged**